external/llvm-project/mlir/lib/Dialect/GPU/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ if (MLIR_ENABLE_ROCM_CONVERSIONS)
     AMDGPUCodeGen
     AMDGPUDesc
     AMDGPUInfo
+    target
```

Comment:
What is this?

Reply:
This is needing to link in libLLVMTarget for having access to the TargetMachine class more fully.